llvm/test/CodeGen/BPF/zext-upper-32bit.ll

```diff
@@ -0,0 +1,34 @@
+; RUN: llc -march=bpfel -mcpu=v3 --filetype=obj < %s | llvm-objdump -d - \
```

Comment:
Matching patterns a bit nicer with objdump, because llc adds some details in the output:

```
test1:                                  # @test1 // want to match
.Ltest1$local:                                   // don't want to match
	.type	.Ltest1$local,@function          // don't want to match
	.cfi_startproc                           // don't want to match
# %bb.0:                                # %entry
	w0 = w1
        exit

```
But you are right, the call to objdump is a bit wasteful. I'll update the test.